app/views/layouts/application.html.haml

```diff
@@ -21,6 +21,7 @@
 
   %body{ class: ("cas" if scotland?) }
     = render "shared/google_tag_manager_no_script"
+    = render "shared/half_star_svg"
```

Comment:
Is having an svg here going to impact screen readers or other assistive technologies? I.e will it require something like `aria-hidden` added to it?

Reply:
Ooh that's a good question, thanks! 

I've just gone through the page using VoiceOver on Firefox, Chrome & Safari. Interestingly, none of them announced the svg earlier in the page where it was rendered, but they behaved differently when they got to the stars in the table. 

Firefox and Chrome say 'image' for every star, which is quite annoying, while Safari announced 'image' for the half-star, but not for any of the other stars.

Adding `aria-hidden` to the half star fixed Safari, but obviously left Chrome & Firefox announcing the full/empty stars but not the half stars. I've therefore added `aria-hidden` to the other star components too, for consistency.